libs/core/src/lib/utils/abstract-fd-ngx-class.ts

```diff
@@ -21,6 +21,14 @@ export abstract class AbstractFdNgxClass implements OnInit, OnChanges {
     /** @hidden */
     abstract _setProperties(): void;
 
+    _setClassToElement(className: string) {
+        (this._elementRef.nativeElement as HTMLElement).classList.value = `${className} ${this.class}`;
+    }
+
+    _clearElementClass() {
```

Comment:
Return type and shouldn't it be named `clearElementClass` (no prefix)?

Reply:
return type added, regarding to `_` same reason as above